client/layout/index.jsx

```diff
@@ -427,7 +429,7 @@ class Layout extends Component {
 				{ config.isEnabled( 'legal-updates-banner' ) && (
 					<AsyncLoad require="calypso/blocks/legal-updates-banner" placeholder={ null } />
 				) }
-				<GlobalNotifications />
+				{ ! shouldDisableGlobalNotifications && <GlobalNotifications /> }
```

Comment:
I think this PR can be a one-liner if we do:

```suggestion
				{ ! isA8CForAgencies() && <GlobalNotifications /> }
```

See my comment below for info on why a prop is unnecessary.

Reply:
Thank you! Sounds good. My initial thought that we might use it somewhere else as well (for example Jetpack Manage). But I'm ok to move it as suggested.